test/e2e_node/podresources_test.go

```diff
@@ -568,6 +571,48 @@ func podresourcesGetAllocatableResourcesTests(ctx context.Context, cli kubeletpo
 	}
 }
 
+func podresourcesGetTests(ctx context.Context, f *framework.Framework, cli kubeletpodresourcesv1.PodResourcesListerClient) {
+	var err error
+	ginkgo.By("checking the output when no pods are present")
+	_, err = cli.Get(ctx, &kubeletpodresourcesv1.GetPodResourcesRequest{PodName: "test", PodNamespace: f.Namespace.Name})
+	framework.ExpectError(err, "pod not found")
+
+	tpd := newTestPodData()
+	ginkgo.By("checking the output when only pods which don't require resources are present")
+	expected := []podDesc{
+		{
+			podName: "pod-00",
+			cntName: "cnt-00",
+		},
+	}
+	tpd.createPodsForTest(ctx, f, expected)
+	resp, err := cli.Get(ctx, &kubeletpodresourcesv1.GetPodResourcesRequest{PodName: "pod-00", PodNamespace: f.Namespace.Name})
+	framework.ExpectNoError(err, "Get() call failed for pod %s/%s", f.Namespace.Name, "pod-00")
+	podResourceList := []*kubeletpodresourcesv1.PodResources{resp.GetPodResources()}
+	res := convertToMap(podResourceList)
+	err = matchPodDescWithResources(expected, res)
+	framework.ExpectNoError(err, "matchPodDescWithResources() failed err %v", err)
+	tpd.deletePodsForTest(ctx, f)
+
+	tpd = newTestPodData()
+	ginkgo.By("checking the output when only pod require CPU")
+	expected = []podDesc{
+		{
+			podName:    "pod-01",
+			cntName:    "cnt-00",
+			cpuRequest: 2000,
```

Comment:
this is a burstable pod. Do you want also to check a guaranteed pod? (if so you need to set requests=limits, or just set limits and the system will automatically add requests=limits)

Reply:
can you point me to example on how to check a guaranteed pod? where should I set the limits?

Reply:
see it already been set here https://github.com/kubernetes/kubernetes/blob/master/test/e2e_node/podresources_test.go#L99-L100.
So it seem it doing already what you want. Isn't it?

Reply:
yes, right, we should be good.